README.md

```diff
@@ -80,6 +80,10 @@ We've provided some very, very basic recipes and are interested in adding more,
 so please send a pull request if you think anything would serve the larger
 community.
 
+## More information
```

Comment:
This should probably be called "Deploying Satellite", and it should have a bit more body so that users can easily find it.